From 79f23e2893b792c01832d74aee261aa4eedb90f6 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Tue, 24 Apr 2018 11:08:41 +0200
Subject: [PATCH 0587/1051] phy: add pcie phy mode

Add PCIe PHY mode to the phy_mode enum to allow configuring generic PHYs
to PCIe by using set_mode callback.

Change-Id: I3b5bdfc35e79a455dca94c735811b9de7f7d3ce2
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 include/linux/phy/phy.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index 9713aebdd348..a1480d76b800 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -40,6 +40,7 @@ enum phy_mode {
 	PHY_MODE_10GKR,
 	PHY_MODE_UFS_HS_A,
 	PHY_MODE_UFS_HS_B,
+	PHY_MODE_PCIE,
 };
 
 /**
-- 
2.17.1

